src/odemis/driver/test/xt_client_test.py

```diff
@@ -43,10 +43,10 @@
                 "inverted": ["x"],
                 }
 CONFIG_FOCUS = {"name": "focuser", "role": "ebeam-focus"}
-CONFIG_SEM = {"name": "sem", "role": "sem", "address": "PYRO:Microscope@192.168.31.130:4242",
+CONFIG_SEM = {"name": "sem", "role": "sem", "address": "PYRO:Microscope@192.168.10.2:4242",
               "children": {"scanner": CONFIG_SCANNER,
                            "focus": CONFIG_FOCUS,
-                           "stage": CONFIG_STAGE,
+                           # "stage": CONFIG_STAGE,
```

Comment:
I will remove these changes as soon as I have performed the last testing (after a first round of review)

Reply:
Still don't push these changes please. These things get easily forgotten to change back.

Reply:
I get your comment, and I will watch it closely. Testing the code is this way much easier since I can just fetch the exact code I am working in.